### ISSUE TITLE
ignoreDuplicates is not working

### ISSUE THREAD

**Issue by AlirezaSN** [SwiftKickMobile/SwiftMessages#328]
Hi, i use `ignoreDuplicates` property but when i call my function 10 times in a short time, just one Message appears. where did i make a mistake? :)

my call code:
```swift
DispatchQueue.main.async {
      for _ in 10.countableRange {
             Toast.show("Hi", color: .red)
      }
}
```

my function:
```swift
public class func show(_ message: String, color: UIColor,
                           config: SwiftMessages.Config? = nil,
                           completion: (() -> Void)? = nil) {
        let messageView = self.bundleView ?? MessageView.viewFromNib(layout: .cardView)
        let bgView = messageView.backgroundView ?? messageView
        bgView.backgroundColor = color
        messageView.configureDropShadow()
        messageView.configureContent(body: message)
        //messageView.customId = Toast.currentFullDateTimeString
        if let font = Helper.shared.fonts.first {
            messageView.bodyLabel?.font = font.withSize(15.0)
        }
        var conf = config ?? SwiftMessages.defaultConfig
        conf.presentationContext = .window(windowLevel: UIWindow.Level.normal)
        conf.eventListeners.append { event in
            if event == .didHide { completion?() }
        }
        conf.ignoreDuplicates = false
        SwiftMessages.show(config: conf, view: messageView)
}
```

meanwhile, the `customId` property is private and we can't set a dynamic string.

thanks a lot.

**Comment by wtmoose** [comment]
Its a bug. Thanks for letting me know.

I guess that feature isn't used much and nobody noticed until you did. If you're in a hurry, `MessageView.id` is writable.

**Comment by AlirezaSN** [comment]
i'll check it. thank you 👍👍

**Comment by wtmoose** [comment]
This is a bug, so leaving open

**Comment by wtmoose** [comment]
Should be fixed in 8.0.3
